software_training_assignment/src/killTurtles.h

```diff
@@ -0,0 +1,8 @@
+#ifndef KILLTURTLES_H
```

Comment:
should just use `#pragma once` instead of header guards, its standard for most compilers